seodongheee2/controllers/postController.js

```diff
@@ -0,0 +1,29 @@
+const postService = require('../services/postServices');
+
+const createPost = async (req, res) => {
+  try {
+    const { title, content,user_id,posting_image_url } = req.body;
```

Comment:
스네이크케이스가 아닌 카멜케이스로 진행해 주세요.
```suggestion
    const { title, content,userId,postingImageUrl } = req.body;
```